src/Elastic.Apm/Model/DbSpanCommon.cs

```diff
@@ -2,17 +2,33 @@
 using System.Data;
 using Elastic.Apm.Api;
 using Elastic.Apm.Helpers;
+using Elastic.Apm.Logging;
 
 namespace Elastic.Apm.Model
 {
 	internal static class DbSpanCommon
 	{
+		internal static class DefaultPorts
+		{
+			internal const int MsSql = 1433;
+			internal const int Oracle = 1521;
+			internal const int MySql = 3306;
+			internal const int PostgreSql = 5432;
+		}
+
 		internal static Span StartSpan(IApmAgent agent, IDbCommand dbCommand) =>
 			(Span)ExecutionSegmentCommon.GetCurrentExecutionSegment(agent).StartSpan(dbCommand.CommandText.Replace(Environment.NewLine, " ")
 				, ApiConstants.TypeDb);
 
-		internal static void EndSpan(Span span, IDbCommand dbCommand, TimeSpan? duration = null)
+		internal static void EndSpan(Span span, IDbCommand dbCommand, IApmLogger logger, DbConnectionStringParser dbConnectionStringParser
```

Comment:
Why we need a logger there if we aren't using it?

Reply:
You are right - we don't need to pass a logger here. I'll fix it.